pip/commands/install.py

```diff
@@ -266,6 +276,7 @@ def run(self, options, args):
                     "continue."
                 )
             install_options.append('--home=' + temp_target_dir)
+            install_options.append('--prefix=')
```

Comment:
This branch should be dead because of exception above. Filled https://github.com/pypa/pip/pull/4132